src/components/ble/NimbleController.cpp

```diff
@@ -138,6 +153,12 @@ void NimbleController::StartAdvertising() {
   fields.uuids128_is_complete = 1;
   fields.tx_pwr_lvl = BLE_HS_ADV_TX_PWR_LVL_AUTO;
 
+  /* The advertising payload is split into advertising data fields
+   * and advertising response fields because all of the data cannot
+   * fit into single packet. Therefore the name of device is sent as
+   * a response to the scan request.
+   */
+
```

Comment:
Should be `/**` and without the ` ` after the `*/` to become a Javadoc-style comment that tooling can pick up.

Reply:
The comment language was copied from the nimble example code. I made a couple of editorial changes and relocated it closer to the code it is describing in this PR. The advertising payload and scan response is clearly documented in the BLE spec so I think the comment is unnecessary (it is still in the nimble example code).